mlops/parallelm/mlops/mlops_ctx.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, config, mode=None):
         self._agents_dict = {}
         self._rest_helper = MlOpsRestFactory().get_rest_helper(self._mode, self._ci.mlops_server, self._ci.mlops_port, self._ci.token)
 
-        if self._mode == MLOpsMode.AGENT:
+        if self._mode == MLOpsMode.AGENT or self._mode == MLOpsMode.REST_ACCUMULATOR:
```

Comment:
It worth to create a dedicated function to test an agent mode case. Who known, maybe in the future we'll have more channel types that work in agent mode.

Reply:
i am skipping this for the time being